blackbox/docs/general/builtins/window-functions.rst

```diff
@@ -23,6 +23,36 @@ into a single row.
 Window Definition
 =================
 
+.. _over:
+
+OVER
+----
+
+Synopsis
+........
+
+::
+
+   OVER (
+      [ PARTITION BY expression [, ...] ]
+      [ ORDER BY expression [ ASC | DESC ] [ NULLS { FIRST | LAST } ] [, ...] ]
+      [ RANGE BETWEEN frame_start AND frame_end ]
+   )
+
+where ``frame_start`` and ``frame_end`` can be one of
+
+::
+
+   UNBOUNDED PRECEDING
+   CURRENT ROW
+   UNBOUNDED FOLLOWING
+
+The default frame definition is ``RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT
+ROW``. If ``frame_end`` is omitted it defaults to ``CURRENT ROW``.
+
+``frame_start`` cannot be ``UNBOUNDED FOLLOWING`` and ``frame_end`` cannot be
+``UNBOUNDED PRECEDING``.
+
```

Comment:
@joemoe @lukas-bernert any thoughts on the documentation for unbounded following windows?

Reply:
lgtm